clients/POSTGRESQL/pg.js

```diff
@@ -8,6 +8,31 @@ module.exports = {
     port: 'port',
     'ca.crt': { ssl: 'ca' },
     'tls.key': { ssl: 'key' },
-    'tls.crt': { ssl: 'cert' }
+    'tls.crt': { ssl: 'cert' },
+    'root.crt': { ssl: 'ca' }
+  },
+  transform: (binding) => {
+    if (binding.options) {
+      binding.options = buildOptionParam(binding.options);
+    }
   }
 };
+
+function buildOptionParam(bindOpt) {
+  // expected raw options format: key1=val1&key2=val2...
+  const optList = [];
+  bindOpt.split('&')
+    .map((item) => item.split('='))
+    .filter(([key, value]) => {
+      return key && value;
+    })
+    .forEach(function ([key, value]) {
+      if (key === '--cluster') {
```

Comment:
Should this be more general. ie any option that starts with `--` should be `--x=y` and anything that does not start with `--` should be `-c x=y`

Reply:
ok if --cluster is unique and all other options have the -c then it makes a bit more sense to me. 

Could you add some additional comments in the code indicating that --cluster is a special case (and why if possible) and that all other options need the `-c`